operator/pkg/controllers/master/certificates.go

```diff
@@ -79,7 +79,9 @@ func kubeAPIServerCertConfig(hostname string, nn types.NamespacedName) *secrets.
 			CommonName: "kube-apiserver",
 			AltNames: certutil.AltNames{
 				DNSNames: []string{hostname, "localhost", "kubernetes", "kubernetes.default",
-					"kubernetes.default.svc", "kubernetes.default.svc.cluster.local"},
+					"kubernetes.default.svc", "kubernetes.default.svc.cluster.local",
+					fmt.Sprintf("%s-cp.%s.svc.cluster.local", nn.Name, nn.Namespace),
```

Comment:
Can you explain why this is needed?

Reply:
When running tests against the guest cluster we don't need to wait for it to be ready/registered with ELB we can directly talk to the service endpoint.